specs.tex

```diff
@@ -7,26 +7,15 @@ \subsection{Arena}
 \label{spec:arena}
 
 \begin{enumerate}
-  \item The arena floor is an \SI{8.4}{m} $\times$ \SI{8.4}{m} rectangle. The
-        tolerance of these two dimensions is $\pm$ \SI{250}{mm}.
-  \item The floor of the arena is carpeted.
+  \item The arena floor is a \SI{6000}{mm} $\times$ \SI{12000}{mm} rectangle.
```

Comment:
Probably also worth noting there are beacons around the arena which can be used for localization.

Reply:
First pass done, will add full details in a rules update.

Reply:
Fully resolved by amendment 2.